submit/src/oscrc.template

```diff
@@ -0,0 +1,8 @@
+[general]
+apiurl = https://api.opensuse.org
+
+[https://api.opensuse.org]
+user=@OBS_USER@
+pass=@OBS_PASSWORD@
+credentials_mgr_class=osc.credentials.PlaintextConfigFileCredentialsManager
+trusted_prj=YaST:Head openSUSE:Factory
```

Comment:
as first step it looks good. Question is what to do with opensuse only packages and its branches like skelcd-control-openSUSE that needs to be submitted to leap? Can we extend it in future to work on specific branches?

Reply:
Yes, it can extended later. First I'd like to handle the Factory packages (most of the packages) and then handle that few special cases.